### DISCUSSION TITLE
Encoder gear 56 mm 30 mm distance

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#42] [Q&A]
Hi all,
i have a slightly different type of lathe.
It has no gear on main spindle, but 2 gears (56 teeth) under it.
One of them turns at 1:40 ratio to main spindle, the other has ratio 1:4.
Is this sufficient for the recommended incremental encoder 600 steps?
If so, the 1:4 56 teeth gear is 30 mm from the wall, so i could'nt use stl from mockendon (designed for 23 mm distance from wall to gear).

**Comment by kachurovskiy** [comment]
Hi @bestbean, can you please attach a photo of your lathe gears? The standard practice is to achieve 400-800 pulses on the encoder per spindle revolution.

**Comment by bestbean** [comment]
lathe is BV20-L (BV20) 9x22
![BV20spindle_gears](https://user-images.githubusercontent.com/13404673/148839942-742baaaa-4d59-41d8-8679-6b97dc0eb051.png)

**Reply by kachurovskiy** [reply]
Interesting, I guess you could use a 200ppr encoder with the 1:4 gear? If you have trouble modelling such a great in Fusion 360, I could help if you post the drawing with dimensions. Cheers!

**Comment by bestbean** [comment]
It is a 1:4 reduction, the faster gear turns 1 time for 4 spindle revolutions.
14 teeth gear for encoder could be an option to achieve 600 pulses on the encoder per spindle revolution.
But then i can't use the provided stl. 
Recommended 600p Encoder already ordered, should be here in a few days.
Thanks for your help!

**Reply by kachurovskiy** [reply]
You need to consider the depth of the encoder, mine is 40mm - what's the distance between the 1:4 gear and the wall? You also can't have a gear with less than ~42 teeth as it might not be able to mesh without the encoded touching the other gear. Maybe a 2000ppr encoder would do the job - https://www.aliexpress.com/item/32961497880.html? Don't know it's dimensions though.

**Comment by bestbean** [comment]
Distance between 1:4 gear and the wall is 30 mm.

**Reply by kachurovskiy** [reply]
Then we should be able to make a similarly offset-to-the-right 56-tooth gear if you get a 2000 line encoder and it's dimensions!

**Comment by bestbean** [comment]
Thanks!
This 2000p encoder also has 6mm axle, other dimensions probably close or similar to the recommended 600p encoder.

**Reply by kachurovskiy** [reply]
Let me know which model of the encoder you end up ordering and I'll make a gear model for you. We should have some time here because stuff from China takes a little while to arrive. Some models of encoders have their dimensions available online when searching for the model name.

**Comment by bestbean** [comment]
i'm looking for these 2port cable ports in 3.5 grid, for nanoELS PCB.
Where can i buy these, what should i enter in search form?

**Reply by kachurovskiy** [reply]
These terminals are optional, you can just solder directly to the board, but it's of course nicer with the terminals - https://www.aliexpress.com/item/1005002724741195.html

**Reply by bestbean** [reply]
Full service, thanks!
Ordered.

**Comment by bestbean** [comment]
Yes, they have diameter from 38 to 40.
Will order one, and measure.
Thanks for your help!

**Reply by bestbean** [reply]
600p encoder order canceled.
OMRON E6B2CWZ6C 2000p/r ordered. 
CAD data: https://download.ia.omron.com/download/page/E6B2_CWZ6C/OEE
dimensions: [q085_e6b2-c_incremental_rotary_encoder_40_mm_datasheet_de.pdf](https://github.com/kachurovskiy/nanoels/files/7849595/q085_e6b2-c_incremental_rotary_encoder_40_mm_datasheet_de.pdf)

**Reply by kachurovskiy** [reply]
@bestbean this STL should work. You'd want to drill the inner 6mm bore on the lathe after printing and super-glue it onto the encoder shaft (a drop from the top). Using a set screw would make it unbalanced due to the weight. Superglue on the other side has little weight and can be dissolved or heated to remove the gear if ever needed.

[omron-E6B2_CWZ6C-gear-off30mm-thick6mm.zip](https://github.com/kachurovskiy/nanoels/files/7849856/omron-E6B2_CWZ6C-gear-off30mm-thick6mm.zip)

**Reply by bestbean** [reply]
Wow, that was fast!
I don't understand "You'd want to drill the inner 6mm bore on the lathe".
I'm understanding that the gear conus should glued to the encoders axle, not screwed.

**Reply by bestbean** [reply]
will one of the 2 proposed encoders base mounts fit this 40mm OMRON thing?

**Reply by kachurovskiy** [reply]
3d printed part would have 6mm bore but it wouldn't be accurate enough, it will likely be undersized by 0.1-0.2mm and not perfectly straight. Drilling it out will improve concentricity and simplify attaching it to the encoder.

**Reply by bestbean** [reply]
Ah ok, thanks.

**Reply by bestbean** [reply]
I managed to resize mockendon's encoder base to 40mm inner diameter (in tinkercad, omg).

**Reply by kachurovskiy** [reply]
If you have a ~60mm round stock, you could just turn this encoder base on your lathe :)

**Reply by bestbean** [reply]
True. I have a 60 mm POM round.
But i 'll prefer to have holes as a option to screw it on.

**Reply by bestbean** [reply]
Wisamic Rotary Encoder 600p/r also has 40 mm diameter.
So, the proposed encoder mount stl will fit the OMRON 2000 p/r encoder.

**Reply by bestbean** [reply]
hole for fixing screw could also be at the bottom.
So, i'll turn the encoder base from POM.
Thanks for this hint!

**Reply by bestbean** [reply]
Encoder gear ordered.

**Comment by bestbean** [comment]
Encoder gear stl worked fine, thanks!
But: The lathe gear used to drive the encoder (red arrow) has huge backslash, about 45 degree spindle rotation.
If this is not tolerable (what i fear), i could use the 42 mm diameter sleeve between belt pulley and spindle back bearing (blue arrow) for some kind of belt drive.
![Getriebe2](https://user-images.githubusercontent.com/13404673/153215769-c2222c79-f36d-4410-8fe3-a410bc34350b.jpg)
Gear on lead screw (green arrow) has 42 teeth.

**Reply by kachurovskiy** [reply]
Got it, let us know if you need any help.

**Comment by bestbean** [comment]
What is the distance between stepper and lead screw shafts at WM210 using the proposed stepper mount?

**Reply by kachurovskiy** [reply]
It's 50mm on centers IIRC

**Reply by bestbean** [reply]
Thank you. So i try to achieve this distance on my lathe, and hopefully can use your stl-files for HTD 5M belt connection with 180 mm belt as proposed for WM210 lathe.
